packages/aws-cdk-lib/aws-certificatemanager/test/dns-validated-certificate.test.ts

```diff
@@ -41,7 +41,7 @@ testDeprecated('creates CloudFormation Custom Resource @aws-cdk/aws-lambda:creat
   });
   Template.fromStack(stack).hasResourceProperties('AWS::Lambda::Function', {
     Handler: 'index.certificateRequestHandler',
-    Runtime: 'nodejs22.x',
+    // Runtime: 'nodejs22.x',
```

Comment:
What is the rationale for this change? If it needs to be removed, let's remove it instead of commenting it out

Reply:
Also, for future reference, it's best not to push changes to already-approved PRs :-) 

You can always create a new PR for the change/fix.